EC_process/timeline.md

```diff
@@ -0,0 +1,54 @@
+# Executive Council Yearly Timeline
+
+Additional tasks that don't fit into the monthly timeline below include:
+* in-person (face-to-face) meeting
+* reassessing the strategic plan
+* annual finance audit
+
+## January
+* Review previous year’s minutes. Make sure all are approved and posted. 
+* Overlap meeting of old/new board
+* Logistics of offboarding outgoing members
+* Logistics of onboarding new members
+
+## February
+* New EC officially takes over
+* Reassess meeting procedures, including methods for setting the agenda and taking minutes
+* Previous year’s financial report for new EC
+* Select new officers
+
+## March 
+
+## April
+* 1st quarter financial report
+
+## May
+
+## June
+
+## July
+* 2nd quarter financial report
+
+## August
+
+## September
+* reassess bylaws (required in even-numbered years, technically due in November)
+
+## October
+* 3rd quarter financial report
+* Annual financial report for community
```

Comment:
Why this late? Isn't that normally a spring thing? Or is the fiscal year not starting January 1st?

Reply:
@tracykteal when is the begin and end of our financial year? As organisation can choose their date in the US. Something for me to add to the Treasurer handbook.

Reply:
I'm not sure how this ended up in October, but suspect it was a part of discussion at our in person meeting last June. 

I've moved release of the annual (calendar year) financial report to March, which would follow review of the report by the new EC in February. This parallels the language in [this blog post](https://carpentries.org/blog/2019/02/projects-teams-job-plans/) from February.

Reply:
The begin of our financial year is January and end is December. Last year, since we'd just started as The Carpentries, we did the financial report later in the year. We were aiming for March this year, but it will be April (in just about a week).